open_feature/provider/no_op_provider.py

```diff
@@ -30,7 +31,7 @@ def get_string_details(
         self,
         key: str,
         default_value: str,
-        evaluation_context: typing.Any = None,
+        evaluation_context: EvaluationContext = EvaluationContext(),
```

Comment:
Looks like there are still a few cases here that need reverting back to `evaluation_context: EvaluationContext = None` ?

Reply:
ok done